### ISSUE TITLE
[JENKINS-75820] Breadcrumb items are not conform to a11y standards

### ISSUE THREAD

**Issue by lemeurherve** [lemeurherve-org/demo#577]
<p><a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility" class="external-link" target="_blank" rel="nofollow noopener">https://developer.mozilla.org/en-US/docs/Web/Accessibility</a> is pointing out the base standard to reach navigation elements. The breadcrumb however is not reachable using only the keyboard which violates the rules outline in the a11y standards revering that the dropdown component is never shown hence there is no way to access the items within</p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/targetx.png" title="targetx's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=targetx">targetx</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-75820" target="_blank">Breadcrumb items are not conform to a11y standards</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/targetx.png" title="targetx's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=targetx">targetx</a>
<li><b>status</b>: In Review
<li><b>priority</b>: Minor
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-14
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p><a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility" class="external-link" target="_blank" rel="nofollow noopener">https://developer.mozilla.org/en-US/docs/Web/Accessibility</a> is pointing out the base standard to reach navigation elements. The breadcrumb however is not reachable using only the keyboard which violates the rules outline in the a11y standards revering that the dropdown component is never shown hence there is no way to access the items within</p></pre>
</details>
</details>
